utils/node-fuzzer/src/runtime/block.rs

```diff
@@ -0,0 +1,92 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2023 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use codec::Encode;
+use frame_support::traits::{OnFinalize, OnInitialize};
+use frame_system::pallet_prelude::BlockNumberFor;
+use gear_common::QueueRunner;
+use gear_runtime::{Authorship, Gear, GearGas, GearMessenger, Runtime, System};
+use pallet_gear::BlockGasLimitOf;
+use sp_consensus_babe::{
+    digests::{PreDigest, SecondaryPlainPreDigest},
+    BABE_ENGINE_ID,
+};
+use sp_consensus_slots::Slot;
+use sp_runtime::{Digest, DigestItem};
+
+/// This is set to some big constant, instead of setting to `BlockGasLimitOf::<Runtime>::get`
+/// because of the known possible dead-lock for the message in the queue, when it's valid gas
+/// limit is more than maximum possible gas rest for the queue execution.
+pub const DEFAULT_GAS_LIMIT: u64 = 240_000_000_000;
```

Comment:
use Perbill::from(90) * BlockGasLimitOf

we can use it with vara runtime, so this gas limit will be pretty low

Reply:
furthermore, fuzzer should random gas limit as well to catch some lower corner cases

Reply:
will state in todo

Reply:
done using `Perbill`